lib/src/key/database/fc.rs

```diff
@@ -54,25 +54,31 @@ mod tests {
 		use super::*;
 		#[rustfmt::skip]
 		let val = Fc::new(
-			"testns",
-			"testdb",
+			1,
+			2,
 			"testfc",
 		);
 		let enc = Fc::encode(&val).unwrap();
-		assert_eq!(enc, b"/*testns\x00*testdb\x00!fntestfc\x00");
+		assert_eq!(
+			enc,
+			vec![
+				b'/', b'*', 0, 0, 0, 1u8, b'*', 0, 0, 0, 2u8, b'!', b'f', b'n', b't', b'e', b's',
+				b't', b'f', b'c', 0x00
+			]
+		);
```

Comment:
Why did this change? I think it makes it harder to read. Happy to not make this a blocker on merging, but I prefer the original.

Reply:
Thanks @phughk! Let me check asap. I thought I made this change from necessity, but that could be due to my sleepy head...

Reply:
Awesome @phughk you were absolutely right this change wasn't necessary! Fixed in [41b7d53](https://github.com/surrealdb/surrealdb/pull/2554/commits/41b7d534ad34058bb91f278c5e811b948ded85f5) 🙏